src/app/app.cpp

```diff
@@ -102,7 +102,22 @@ int App::run(int argc, char** argv)
 
     QGuiApplication::styleHints()->setMousePressAndHoldInterval(250);
 
-    QApplication app(argc, argv);
+    // ====================================================
+    // Parse command line options
+    // ====================================================
+    CommandLineController commandLine;
+    commandLine.init();
+    commandLine.parse(argc, argv);
```

Comment:
I believe that now this class needs to be renamed to just `CommandLineParser`, because it no longer performs an action (like a controller)

Reply:
renamed